docs/en/tutorials/extra-lazy-associations.rst

```diff
@@ -19,6 +19,7 @@ can be called without triggering a full load of the collection:
 -  ``Collection#count()``
 -  ``Collection#get($key)``  (available with Doctrine 2.4)
 -  ``Collection#slice($offset, $length = null)``
+-  ``Collection#removeElement($element)`` for either owning side collections or inverse collections with orphanRemoval=true
```

Comment:
With this addition we should be fine closing https://github.com/doctrine/orm/issues/7765. Could you maybe change line below (I can't suggest change there) to mention DELETE query? Something along "For each call, if the Collection is not yet loaded, issue a straight SELECT/DELETE statement against the database."